litestar/cli/_utils.py

```diff
@@ -272,9 +274,22 @@ def _bool_from_env(key: str, default: bool = False) -> bool:
     return value in ("true", "1")
 
 
+def _validate_app_path(app_path: str) -> tuple[ModuleType, str]:
+    try:
+        module_path, app_name = app_path.split(":")
+        module = importlib.import_module(module_path)
+    except ValueError:
+        console.print(f"Invalid argument passed --app {app_path!r}[bold red] Expected 'module:app'")
+        sys.exit(1)
+    except ModuleNotFoundError:
+        console.print(f"Invalid argument passed --app {app_path!r}:[bold red] Module Not Found")
+        sys.exit(1)
+    else:
+        return module, app_name
```

Comment:
```suggestion
    except ValueError:
        console.print(f"Invalid argument passed --app {app_path!r}[bold red] Expected 'module:app'")
        sys.exit(1)
    except ModuleNotFoundError:
        console.print(f"Invalid argument passed --app {app_path!r}:[bold red] Module Not Found")
        sys.exit(1)
        
    module = importlib.import_module(module_path)
    return module, app_name
```
So we don't hide any other errors accidentally :)

Reply:
Thank a lot. I made a correction in the new commit.